webpack.config.babel.js

```diff
@@ -20,7 +20,7 @@ const config = {
 	module: {
 		rules: [
 			{
-				test: /\.jsx?$/,
+				test: /\.js?$/,
```

Comment:
I never make the extensions jsx.. I think it's pointless to label. Otherwise I would also have to do .es6

Feel free to discuss or argue